pylossless/pipeline.py

```diff
@@ -495,37 +451,51 @@ def _detect_outliers(array, flag_dim='epoch', outlier_method='quantile',
         outlier_mask = outlier_mask | (array < l_out)
 
     # average column of outlier_mask
-    prop_outliers = outlier_mask.mean(operate_dim)
-    return np.where(prop_outliers > flag_crit)[0]
+    # drop quantile coord because it is no longer needed
+    prop_outliers = outlier_mask.astype(float).mean(operate_dim)
+    if "quantile" in list(prop_outliers.coords.keys()):
+        prop_outliers = prop_outliers.drop_vars('quantile')
+    return prop_outliers[prop_outliers > flag_crit].coords.to_index().values
 
 
-def add_pylossless_annotations(raw, inds, event_type, epochs):
-    """Add annotations for flagged epochs.
+def _threshold_volt_std(epochs, flag_dim, threshold=5e-5):
```

Comment:
Private utility function we created to assess channel/epoch criteria based on a fixed threshold (of the voltage-var stdev). Review If You would like to.